src/DynamoCoreWpf/Controls/NodeAutoCompleteSearchControl.xaml.cs

```diff
@@ -37,21 +37,9 @@ public NodeAutoCompleteSearchControl()
             {
                 Application.Current.Deactivated += currentApplicationDeactivated;
             }
-            Loaded += NodeAutoCompleteSearchControl_Loaded;
             Unloaded += NodeAutoCompleteSearchControl_Unloaded;
         }
 
-        private void NodeAutoCompleteSearchControl_Loaded(object sender, RoutedEventArgs e)
-        {
-            if (ViewModel != null && ViewModel.PortViewModel != null)
-            {
-                ViewModel.PortViewModel.PlaceNodeAutocompleteWindow(this, e);
-                Analytics.TrackEvent(
-                Dynamo.Logging.Actions.Open,
-                Dynamo.Logging.Categories.NodeAutoCompleteOperations);
```

Comment:
@aparajit-pratap @zeusongit I'm fine removing the loaded function but we need to move the analytics call to a different place

Reply:
I noticed an issue with this loaded function. It wasn't getting called for some reason after opening a new workspace, which is why I decided to remove it. I didn't realize analytics info was added here. Sorry for overlooking that.

Reply:
@aparajit-pratap @zeusongit No problem. There is a show/hide place of code we can move to, see `OnNodeAutoCompleteSearchControlVisibilityChanged()`. May need to do the same change for in-canvas search as well.. if this loaded function is not reliable, because @zeusongit added that for in canvas search following your example here

Reply:
@zeusongit what event exactly does the `analytics.trackevent` aim to track here - the autocomplete popup popping up?

Reply:
@aparajit-pratap Yes, so that we know how frequent users using this comparing to in canvas search so we can make improvements